usrp3/tools/scripts/uhd_image_builder.py

```diff
@@ -21,6 +21,7 @@
 import os
 import re
 import glob
+import yaml
```

Comment:
This could fail, because yaml is not a dependency.

Reply:
Agreed. That was one area of concern.

A potential solution could be to import yaml only in the yaml reader function, wrapped with a try/except block to fail gracefully. This way, if you don't have the yaml module installed, you only lose the yaml functionality (not crash the program).

Would that be ok?

Reply:
... Ill also update with this fix tomorrow too to see if it looks good

Reply:
That would be perfect.